src/pages/Home.jsx

```diff
@@ -19,73 +59,79 @@ function Home() {
                 <li className="nav-item">
                   <button
                     type="button"
-                    className="nav-link" // Add active if not feed and not tag
+                    onClick={()=> setActiveTag(null)}
+                    className={classNames("nav-link", {active: !activeTag})}
                   >
                     Global Feed
                   </button>
                 </li>
                 <li className="nav-item">
-                  <a className="nav-link active"># {faker.lorem.word()}</a>
+                 {activeTag && <a className="nav-link active"># {activeTag}</a>}
                 </li>
               </ul>
             </div>
+            {articlesQuery.isFetching && <div className="article-preview">Loading articles..</div>}
+            {(articles?.length === 0 && articlesQuery.isFetched) && <div className="article-preview">No articles to show... Wanna be the first to post?</div>}
```

Comment:
Nice touch! A few small things:

- Let's move this to the top before we try and render any articles. 
- Let's use `isEmpty` from 'lodash-es` to check that the article list is empty.

Reply:
Fixed, please check